package-dev/Runtime/SentryIntegrations.cs

```diff
@@ -0,0 +1,63 @@
+#if UNITY_2020 || UNITY_2021 || UNITY_2022
+#define SENTRY_SCENE_MANAGER_TRACING_INTEGRATION
+#endif
+
+using System;
+using Sentry.Integrations;
+using UnityEngine;
+using UnityEngine.Scripting;
+using UnityEngine.SceneManagement;
+
+namespace Sentry.Unity
+{
+    public static class SentryIntegrations
+    {
+        public static void Configure(SentryUnityOptions options)
+        {
+#if SENTRY_SCENE_MANAGER_TRACING_INTEGRATION
+            if (options.TracesSampleRate > 0.0) {
+                options.AddIntegration(new SceneManagerTracingIntegration());
+            } else {
+                options.DiagnosticLogger?.Log(SentryLevel.Debug,
+                    "Skipping SceneManagerTracing integration because performance tracing is disabled.");
+            }
+#endif
+        }
+    }
+
+#if SENTRY_SCENE_MANAGER_TRACING_INTEGRATION
+    internal class SceneManagerTracingIntegration : ISdkIntegration
+    {
+        public void Register(IHub hub, SentryOptions options)
+        {
+            if (SceneManagerAPI.overrideAPI is not null)
```

Comment:
I was thinking the same at first but no, we can't because `LoadSceneAsyncByNameOrIndex` is protected so we can't call it on the existing `SceneManagerAPI.overrideAPI` from our implementation.